pages/portfolio.html

```diff
@@ -4,9 +4,59 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>My Work</title>
+  <link href="/stylesheets/style.css" rel="stylesheet">
+  <link href="/stylesheets/portfolio-style.css" rel="stylesheet">
 </head>
+
 <body>
+  <main class="container">
+    <header><h1>tiffini hyatt</h1></header>
+
+    <nav>
+      <menu class="nav-links">
+        <li id="home"><a href="pages/index.html"><h4>home</h4></a></li>
+        <li id="portfolio"><a href="pages/portfolio.html"><h4>work</h4></a></li>
+        <li id="about"><a href="pages/about.html"><h4>about me</h4></a></li>
+      </menu>
+    </nav>
+
+    <div class="projects">
+      <div id="project-1">
```

Comment:
Rather than `project-n` for the `id` names on these project blocks, I'd consider using a name that reflects the project itself like `to-do-project`or `to-do-list-description` for this particular one. Thinking about this in the context of a larger, long-term project, if you had to re-order these someday, then the numbering in the `id`s may not make sense in the new order and introduce confusion for other team mates as to what the numbering represents.